examples/auth/envoy_config.json

```diff
@@ -338,10 +338,6 @@
                               "timeout": "15s"
                             },
                             "typedPerFilterConfig": {
-                              "com.google.espv2.filters.http.service_control": {
```

Comment:
Why is service_control perRouteConfig removed?

Reply:
Because we don't set service control filter in this test and before the refactor, the route perRouteConfig isn't aware of that the service control filter is missing.

https://github.com/GoogleCloudPlatform/esp-v2/blob/1ffe51f3e7526214e050f6eff9084ed3656e1d05/src/go/bootstrap/static/bootstrap_test.go#L61